### PR TITLE
Update the commands in the plutus-pab README.md

### PR DESCRIPTION
**Pull request by koslambrou**
As @james-iohk pointed out, the commands in the plutus-pab README were out of date. This PR updates them.

Pre-submit checklist:
- Branch
    - [x] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

File: plutus-pab/README.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Generates the purescript bridge code.
 ### migrate
 
 ```
-$ cabal run plutus-pab -- migrate
+$ cabal exec plutus-pab-examples -- migrate --config plutus-pab/plutus-pab.yaml.sample
```
Comment (james-iohk) [inline plutus-pab/README.md:104]: Seeing as the readme is in `plutus-pab` directory then config files should be relative to here.
Change this and other occurrences below to `--config plutus-pab.yaml.sample`

**Review by james-iohk — APPROVED** [review]
Thanks

File: plutus-pab/README.md (hunk 1 above)
Comment (koslambrou) [inline plutus-pab/README.md:104]:
```suggestion
$ cabal exec plutus-pab-examples -- migrate --config plutus-pab.yaml.sample
```

File: plutus-pab/README.md [hunk 2]
```diff
@@ -115,25 +115,22 @@ Migrates the database in `pab-core.db` to the current schema.  The database cont
 ### all-servers
 
 ```
-$ cabal run plutus-pab -- all-servers
+$ cabal exec plutus-pab-examples -- all-servers --config plutus-pab/plutus-pab.yaml.sample
```
Comment (koslambrou) [inline plutus-pab/README.md:118]:
```suggestion
$ cabal exec plutus-pab-examples -- all-servers --config plutus-pab.yaml.sample
```

File: plutus-pab/README.md [hunk 3]
```diff
@@ -170,7 +164,7 @@ Starts all mocks and core services *except for* the mock node service:
 ### wallet-server
 
 ```
-$ cabal run plutus-pab -- wallet-server
+$ cabal exec plutus-pab-examples -- wallet-server --config plutus-pab/plutus-pab.yaml.sample
```
Comment (koslambrou) [inline plutus-pab/README.md:167]:
```suggestion
$ cabal exec plutus-pab-examples -- wallet-server --config plutus-pab.yaml.sample
```

File: plutus-pab/README.md [hunk 4]
```diff
@@ -183,18 +177,26 @@ Plutus specific wallet implementation for managing user funds on the blockchain.
...
+- [Cardano.Wallet.LocalClient](https://github.com/input-output-hk/plutus-apps/blob/main/plutus-pab/src/Cardano/Wallet/LocalClient.hs)
+
+Remote wallet:
+- [Cardano.Wallet.RemoteClient](https://github.com/input-output-hk/plutus-apps/blob/main/plutus-pab/src/Cardano/Wallet/RemoteClient.hs)
 
 ### webserver
 
 ```
-$ cabal run plutus-pab -- webserver
+$ cabal exec plutus-pab-examples -- webserver --config plutus-pab/plutus-pab.yaml.sample
```
Comment (koslambrou) [inline plutus-pab/README.md:199]:
```suggestion
$ cabal exec plutus-pab-examples -- webserver --config plutus-pab.yaml.sample
```

File: plutus-pab/README.md [hunk 5]
```diff
@@ -215,16 +217,16 @@ Serves the PAB user interface
 ### node-server
 
 ```
-$ cabal run plutus-pab -- node-server
+$ cabal exec plutus-pab-examples -- node-server --config plutus-pab/plutus-pab.yaml.sample
```
Comment (koslambrou) [inline plutus-pab/README.md:220]:
```suggestion
$ cabal exec plutus-pab-examples -- node-server --config plutus-pab.yaml.sample
```

File: plutus-pab/README.md [hunk 6]
```diff
@@ -238,7 +240,7 @@ Mock-implementation of a Goguen node. Clients to this service are:
 ### chain-index
 
 ```
-$ cabal run plutus-pab -- chain-index
+$ cabal exec plutus-pab-examples -- chain-index --config plutus-pab/plutus-pab.yaml.sample
```
Comment (koslambrou) [inline plutus-pab/README.md:243]:
```suggestion
$ cabal exec plutus-pab-examples -- chain-index --config plutus-pab.yaml.sample
```

File: plutus-pab/README.md [hunk 7]
```diff
@@ -143,24 +140,21 @@ Combines the execution of all core services and mocks in the appropriate order:
 ### client-services
 
 ```
-$ cabal run plutus-pab -- client-services
+$ cabal exec plutus-pab-examples -- client-services --config plutus-pab/plutus-pab.yaml.sample
```
Comment (koslambrou) [inline plutus-pab/README.md:143]:
```suggestion
$ cabal exec plutus-pab-examples -- client-services --config plutus-pab.yaml.sample
```

**Pull request merged**